src/modules/products/entities/product.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { User } from '../../user/entities/user.entity';
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, Entity, ManyToOne } from 'typeorm';
+
+@Entity()
+export class Product extends AbstractBaseEntity {
+  @ApiProperty()
```

Comment:
This decorator should be on the Dto which is the primary interface for exchanging data between the client and the API. It's not meant to be on the entity